src/features/trivia/index.ts

```diff
@@ -92,15 +93,27 @@ export async function sendTriviaQuestion(guild: Guild) {
     collector.on("end", (_, reason) => {
       if (reason !== winnerStopReason) {
         resetPointsToGive();
```

Comment:
This will reset `pointsToGive` before the conditional on the next line. I reckon we move this to the bottom.